warehouse/internal/repo/table_upload.go

```diff
@@ -301,7 +301,8 @@ func (repo *TableUploads) Set(ctx context.Context, uploadId int64, tableName str
 	}
 	if options.Error != nil {
 		setQuery.WriteString(fmt.Sprintf(`error = $%d,`, len(queryArgs)+1))
-		queryArgs = append(queryArgs, *options.Error)
+		sanitizedError := warehouseutils.SanitizeString(*options.Error)
```

Comment:
We were getting [`unable to change upload columns: pq: unsupported Unicode escape sequence`](https://github.com/rudderlabs/rudder-server/pull/3135/files#r1151081530) error.